schedview/dayobs.py

```diff
@@ -1,16 +1,26 @@
 import copy
 import datetime
 from dataclasses import dataclass
-from functools import cached_property
+from functools import cached_property, partial
 from typing import Literal, Self
 
+import astropy.units as u
 import dateutil.parser
+import numpy as np
+import scipy.optimize
+from astropy.coordinates import AltAz, EarthLocation, HADec, SkyCoord, get_body
 from astropy.time import Time
 
 DAYOBS_TZ = datetime.timezone(datetime.timedelta(hours=-12))
 MJD_EPOCH = datetime.date(1858, 11, 17)
 ONE_DAY = datetime.timedelta(days=1)
 
+LSE30_ATMOSPHERE = {
+    "pressure": 750.0 * 100 * u.Pa,
+    "temperature": 11.5 * u.deg_C,
+    "relative_humidity": 0.4,
+}
```

Comment:
The `rubin_scheduler.utils.Site` class, when instantiated for "LSST" should also have this same information. If you need to keep this here separately, that's fine, but consider using the Site class instead so that if we find that a different value is more appropriate for Rubin, we can change it in as few places as possible.

Reply:
I originally used `rubin_scheduler.utils.Site` when I first wrote this, but I thought the goal was to move this somewhere where it wouldn't depend on rubin_scheduler, so I got rid of that dependency by copying it.

Reply:
I also made the modifications to the isclose calls to set rtol and atol to something reasonably demanding.